db/seeds.rb

```diff
@@ -81,8 +81,8 @@
 Task.destroy_all
 Collaboration.destroy_all
 Assignment.destroy_all
-Expenses.destroy_all
-Guests.destroy_all
```

Comment:
Just a note but thse stuff would not work if you run it once. Make sure that you know your edit is working at least on your local machine before you make a PR!

Reply:
Didn't think about that thank you.